rudder-lang/src/error.rs

```diff
@@ -42,6 +42,7 @@ use std::collections::HashMap;
 ///
 use std::fmt;
 use std::hash::Hash;
+use colored::*;
```

Comment:
wildcard use should generally be avoided as it may break in the future in case new items added to the library may collide with existing names. You should be able to just import the required trait for color formatting.